src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -8,6 +8,7 @@
   "org.folio.dao",
   "org.folio.service",
   "org.folio.clients",
-  "org.folio.rest.impl"})
+  "org.folio.rest.impl",
+  "org.folio.rest.exceptions"})
```

Comment:
No need to add _rest.exceptions_, there are no components annotated by @Component / @Service